scripts/ci/release_version_cal.py

```diff
@@ -37,11 +37,11 @@
 def extract_module_history_update_info(mod_update_info, mod):
     """
     re pattern:
-    --- a/src/monitor-control-service/HISTORY.rst
-    +++ b/src/monitor-control-service/HISTORY.rst
+    --- a/src/monitor-control-service/HISTORY.rst/md
+    +++ b/src/monitor-control-service/HISTORY.rst/md
```

Comment:
It looks like you want to match HISTORY.rst or HISTORY.md?
```suggestion
    --- a/src/monitor-control-service/HISTORY.(rst|md)
    +++ b/src/monitor-control-service/HISTORY.(rst|md)
```

Reply:
yes, found HISTORY.md in some extension module